_config.yml

```diff
@@ -2,7 +2,7 @@ title: Just the Docs Template
 description: A starter template for a Jeykll site using the Just the Docs theme!
 theme: just-the-docs
 
-url: https://just-the-docs.github.io
+url: https://yourname.github.io/repo/
```

Comment:
This will affect the actual website that we deploy to showcase this theme; could we instead add a note that users should change this?

Reply:
Oh well! I wasn't aware of that!